jerry-core/include/jerryscript-core.h

```diff
@@ -159,6 +159,18 @@ typedef struct
 } jerry_property_descriptor_t;
 
 /**
+ * Description of JerryScript heap memory stats.
+ * It is for memory profiling.
+ */
+typedef struct
+{
+  size_t version; /**< the version of the stats struct */
```

Comment:
The problem is if you dynamically link Jerry, and you compiled your code with an older Jerry, and the system has a newer one, you will experience a crash. Currently most projects statically link Jerry, so this is not a problem, but it might be if some distros will support it.